### DISCUSSION TITLE
How to schedule Job Batches?

### DISCUSSION THREAD

**Discussion by Quadrubo** [laravel/framework#46086] [Q&A] [answered]
Hello,

I have a Job that pings a computer `PingComputer`. When I need to ping all computers I create a job batch containing multiple instances of this Job.

I also wanna schedule calling all the computers. Is there any way to do this via scheduling? I can't seem to find anything about scheduling job batches. 
Thanks for your help!

**Comment by riotCode** [comment, marked as answer]
I would say write a command that triggers the job batches and add that to the scheduler.

**Reply by Quadrubo** [reply]
Ah thanks that's a great idea don't know why I didn't think of that!

**Reply by Quadrubo** [reply]
this is still kind of a workaround but seems better than creating a job calling many jobs.

**Comment by 1337erik** [comment]
why not just $schedule->call( function(){ Bus::batch([ .... ]); } );
